src/app/components/Video/index.jsx

```diff
@@ -1,30 +1,63 @@
 import React from 'react';
+import styled from 'styled-components';
+import Helmet from 'react-helmet';
 import { videoComponentPropTypes } from '../../models/propTypes';
 
+const MediaPlayerContainer = styled.div`
+  height: 26em;
+  width: 100%;
+`;
+
 const Video = ({
+  id,
   pid,
-  kind,
   title,
-  items,
-  holdingImageUrl,
   statsAppName,
   statsAppType,
   statsCountername,
   statsDestination,
   uiLocale,
+  mediaPlayerSettings,
 }) => (
-  <div>
-    <div>video pid: {pid}</div>
-    <div>kind: {kind}</div>
+  <>
+    <Helmet>
+      <script type="text/javascript">
+        {`
+          function mediaPlayerSetup(container) {
+            require(['bump-4'], (bump) => {
+              var mediaPlayer = bump.player(
+                document.getElementById('mediaPlayer${pid}'),
```

Comment:
Actually, I disagree with how we're constructing the `id`. We should just pass `id` to this component and not have to manipulate it. 

https://github.com/bbc/simorgh/blob/5d0efc0cbad31785efdbc64755e7a0b64c5f587b/src/app/containers/Video/index.jsx#L85-L87

Whatever we pass to this component is what we should use as the ID, no manipulating it here. If we change `id` in the above snippet to `fooBar`, a console error will be thrown because the element you're trying to reference won't exist, and we won't have a media player.